pubtools/_quay/push_docker.py

```diff
@@ -180,7 +162,6 @@ def push_docker(push_items, signing_key, hub, task_id, target_name, target_setti
         stepper.run(start_from=-1)
         raise
     finally:
-        log_push_items(signing_key, items=push_items)
```

Comment:
I am just curios why we do not wan to log  items?

Reply:
we do want to log those, but it will require some changes in pushcollector and pub so basically it's not possible to call update_push_items here as we don't have mapping of push item states available outside of pub yet. So as log_push_items are called only in the begin and at the end of the code we can remove code from here and have it in pub instead